pkgs/test_api/lib/scaffolding.dart

```diff
@@ -16,6 +16,7 @@ export 'src/backend/configuration/skip.dart' show Skip;
 export 'src/backend/configuration/tags.dart' show Tags;
 export 'src/backend/configuration/test_on.dart' show TestOn;
 export 'src/backend/configuration/timeout.dart' show Timeout;
+export 'src/backend/test_location.dart' show TestLocation;
```

Comment:
I don't know if the changes I made to public API are reasonable (or if I've exporting things in the right ways), so please check this carefully :-)

Reply:
the API changes LGTM